4337/contracts/test/TestToken.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol";
+
+contract HariWillibaldToken is ERC20, ERC20Permit {
+    constructor(address initialMintDest) ERC20("Hari Willibald Token", "HWT") ERC20Permit("Hari Willibald Token") {
```

Comment:
Who is "Hari Wallibal"?! 🤣

Reply:
@rmeissner might have more context on this